src/main/menu.js

```diff
@@ -80,16 +81,27 @@ function menuTemplate (ipc) {
             logger.info('[MENU] Import Configuration', result)
             if (result.canceled) return
             if (!result.filePaths || !result.filePaths.length) return
-            userConfig.importSettings(result.filePaths[0], (err) => {
+            userConfig.importSettings(result.filePaths[0], (err) => 
+            {
               if (err) return onerror(err)
-              ipc.send('reload-config', (err) => {
-                if (err) logger.error(err)
-                logger.debug('[SYSTEM] Forcing window refresh')
+              ipc.send('reload-config', async (err) => 
+              {
+                if (err) 
+                {
+                  logger.error(err)
+                  logger.debug('[SYSTEM] Forcing window refresh')
+                  focusedWindow.webContents.send('force-refresh-window')
+                  logger
+                  return
+                }
+                await dialog.showMessageBox({message:t('menu-config-complete'), buttons:[t("button-submit")]})
```

Comment:
think we'd ideally show the name of the config that's been imported (similar to [mobile](https://github.com/digidem/mapeo-mobile/pull/864)). Would require a little bit of work in [`user-config.js`](https://github.com/digidem/mapeo-desktop/blob/660754ea6483064f6f909390d010245a18461bb1/src/main/user-config.js#L7) to pass the settings metadata